### PR TITLE
Nturinski/deploy

### PR DESCRIPTION
**Pull request by nturinski**
Will be broken until https://github.com/Microsoft/vscode-azuretools/pull/33 is merged

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,6 +72,15 @@
                     "dark": "resources/dark/refresh.svg"
                 }
             },
+            {
+                "command": "appService.Deploy",
```
Comment (EricJizbaMSFT) [inline package.json]: Add this to activationEvents

File: package.json [hunk 2]
```diff
@@ -293,44 +307,34 @@
                     "group": "2_appServiceControlCommands@4"
                 },
                 {
-                    "command": "appService.DeployZipPackage",
+                    "command": "appService.OpenVSTSCD",
                     "when": "view == azureAppService && viewItem == appService",
                     "group": "3_appServiceDeployCommands@1"
                 },
-                {
-                    "command": "appService.LocalGitDeploy",
```
Comment (EricJizbaMSFT) [inline package.json]: You can remove `LocalGitDeploy` and `DeployZipPackage` completely correct? They're still in this file a few times

File: src/explorer/SiteTreeItem.ts [hunk 3]
```diff
@@ -109,19 +109,6 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
...
-    public async localGitDeploy(client: WebSiteManagementClient): Promise<void> {
-        const fsWorkspaceFolder = await util.showWorkspaceFoldersQuickPick('Select the folder to Local Git deploy.');
-        try {
-            await this.siteWrapper.localGitDeploy(fsWorkspaceFolder.uri.fsPath, client, getOutputChannel());
-        } catch (err) {
-            if (err instanceof UserCancelledError) {
-                throw err;
-            }
-            const appServicePlan = await getAppServicePlan(this.site, client);
-            throw new SiteActionError(err, appServicePlan.sku.size);
```
Comment (EricJizbaMSFT) [inline src/explorer/SiteTreeItem.ts]: I would add this SiteActionError logic back to the main Deploy command

File: package.json [hunk 4]
```diff
@@ -249,6 +258,11 @@
                     "command": "appService.Refresh",
                     "when": "view == azureAppService",
                     "group": "navigation@2"
+                },
+                {
+                    "command": "appService.Deploy",
+                    "when": "view == azureAppService",
+                    "group": "navigation@3"
```
Comment (EricJizbaMSFT) [inline package.json]: I would put this between create and refresh. Refresh is consistently the last command for all our extensions

File: package.json [hunk 5]
```diff
@@ -2,7 +2,7 @@
     "name": "vscode-azureappservice",
```
Comment (EricJizbaMSFT) [inline package.json]: Not sure if you were just waiting for it to be published, but make sure to update the version of the appservice npm package in this file

File: package.json [hunk 6]
```diff
@@ -293,44 +298,34 @@
                     "group": "2_appServiceControlCommands@4"
                 },
                 {
-                    "command": "appService.DeployZipPackage",
```
Comment (whiskeyjay) [inline package.json]: Why doesn't the App Service node have the deploy command?

Comment (EricJizbaMSFT) [reply package.json]: Per discussion with @fiveisprime we thought this was a confusing entry point and replaced it with the one on the title bar. The node entry point implied you were deploying the app service itself (as opposed to deploying TO the app service)

Comment (whiskeyjay) [reply package.json]: What does it mean by "deploying the app service itself"? The app service is already there, why would someone think the service itself needs to be deployed again?

Comment (whiskeyjay) [reply package.json]: I'll approve for now. We can discuss this a bit more offline.

**Pull request merged**